CMakeLists.txt

```diff
@@ -324,6 +324,7 @@ macro (nng_check_sym SYM HDR DEF)
     check_symbol_exists (${SYM} ${HDR} ${DEF})
     if (${DEF})
         add_definitions (-D${DEF}=1)
+        set (${DEF} ON)
```

Comment:
This should not be needed.  CMake should already have set that with check_symbol_exists.

Reply:
it's needed in order to conditionalize other cmake inclusions, like the one in src/CMakeLists.txt - let me know if you'd prefer another approach.

Reply:
Let me look at this in more detail -- I still think this is redundant unless I've misunderstood what check_symbol_exists does.